docs/SDL/OnAppPermissionConsent/index.md

```diff
@@ -2,16 +2,80 @@
 
 
 ### Notification
+Type
+: Notification  
+Sender
+: HMI  
+Purpose
+: Inform about the User permission changes for some functionality for the named application.
```

Comment:
This formats _incredibly_ oddly on the dev portal, I gather that the `Type`, `Sender`, and `Purpose` lines should be separated by a newline. In addition, the values for each of these keys should be on the same line, not separated.